x/ibc/testing/solomachine.go

```diff
@@ -32,27 +33,31 @@ func NewSolomachine(t *testing.T, clientID string) *Solomachine {
 	privKey := ed25519.GenPrivKey()
 
 	return &Solomachine{
-		t:          t,
-		ClientID:   clientID,
-		PrivateKey: privKey,
-		PublicKey:  privKey.PubKey(),
-		Sequence:   1,
-		Time:       10,
+		t:           t,
+		ClientID:    clientID,
+		PrivateKey:  privKey,
+		PublicKey:   privKey.PubKey(),
+		Sequence:    1,
+		Time:        10,
+		Diversifier: "", // TODO: use custom diversifier?
```

Comment:
not sure what should I use here

Reply:
I'd make this a param in `NewSolomachine` that way we can use `""` and non empty strings in different parts of our testing to ensure both are being tested. We could add to the wish list to make the diversifier generation deterministically random.